class APIError(Exception):
    pass


class UserError(Exception):
    pass
